packages/driver/src/cy/net-stubbing/add-command.ts

```diff
@@ -91,6 +91,13 @@ function isNumberMatcher (obj): obj is NumberMatcher {
   return Array.isArray(obj) ? _.every(obj, _.isNumber) : _.isNumber(obj)
 }
 
+function isMethod (arg: string) {
+  const httpRequestMethods = ['get', 'head', 'post', 'put', 'delete', 'connect', 'options', 'trace', 'patch']
```

Comment:
We have a util in the driver already to check for this: https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress/utils.js#L277:L277

Uses the `method` node module.

Reply:
Done.